### DISCUSSION TITLE
Welcome to uptrain Discussions!

### DISCUSSION THREAD

**Discussion by sourabhagr** [uptrain-ai/uptrain#44] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ArslanKAS** [comment]
Hi there, is there a way to do evaluations without providing any API-Key at all?

I mean is API-Key really required?
